Templates/Common-Backend-Scripts/packageBuildOutputs.sh

```diff
@@ -406,83 +490,93 @@ validatePublishingOptions() {
     fi
 }
 
-# Call validate input options
-if [ $rc -eq 0 ]; then
-    validateOptions
-fi
-
-# compute parameters
-if [ $rc -eq 0 ]; then
-    # Compute artifactRepositoryName based on function in packageBuildOutputs.config
-    artifactRepositoryName=$(computeArtifactRepositoryName)
-    artifactRepositoryDirectory=$(computeArtifactRepositoryDirectory)
+# compute packaging parameters and validate publishing options
+if [ $rc -eq 0 ] && [ "$publish" == "true" ]; then
+    # invoke function in packageUtils
 
-    # set default PipelineType for pipelines on main
-    if [ "${Branch}" == "main" ] && [ -z "${PipelineType}" ]; then
-        PipelineType="build"
+    if [ ! -z "${tarFileName}" ]; then
+        echo $PGM": [INFO] ** Identified that tarFileName is passed into packageBuildOutputs.sh (${tarFileName}). This will be reset and recomputed based on buildIdentifier and releaseIdentifier to align with the conventions for packaging."
+    fi
+    
+    if [ ! -z "${artifactVersionName}" ]; then
+        echo $PGM": [INFO] ** Identified that artifactVersionName is passed into packageBuildOutputs.sh (${artifactVersionName}). This will be reset and recomputed based on buildIdentifier and releaseIdentifier to align with the conventions for packaging."
     fi
+
+    computePackageInformation
```

Comment:
Why not calling the the ArtifactRepositoryHelpers.groovy to compute this, instead of introducing a new utility script (packageUtils.sh)?